<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblOriginal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblProcessed">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPauseResume">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>488</y>
      <width>93</width>
      <height>113</height>
     </rect>
    </property>
    <property name="text">
     <string>pause stream</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtXYRadius">
    <property name="geometry">
     <rect>
      <x>964</x>
      <y>480</y>
      <width>265</width>
      <height>117</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="SldBlueLower">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SldGreenLower">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>536</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SldRedLower">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>572</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SldBlueValue">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>500</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SldGreenValue">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>532</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SldRedValue">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>564</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="SldBlueUpper">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>500</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SldGreenUpper">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>536</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SldRedUpper">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>572</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="B">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>508</y>
      <width>30</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="G">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>540</y>
      <width>30</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLabel" name="R">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>576</y>
      <width>30</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtparam1">
    <property name="geometry">
     <rect>
      <x>824</x>
      <y>500</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtminRadius">
    <property name="geometry">
     <rect>
      <x>824</x>
      <y>568</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtparam2">
    <property name="geometry">
     <rect>
      <x>892</x>
      <y>500</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtmaxRadius">
    <property name="geometry">
     <rect>
      <x>892</x>
      <y>568</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblparam1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>param1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblparam2">
    <property name="geometry">
     <rect>
      <x>904</x>
      <y>480</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>param2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblminRadius">
    <property name="geometry">
     <rect>
      <x>832</x>
      <y>548</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>minRadius</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblmaxRadius">
    <property name="geometry">
     <rect>
      <x>896</x>
      <y>544</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>maxRadius</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtLoopTime">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>568</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>548</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>set loop time [ms]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
